src/jfr/jenkinsfile-runner.inc

```diff
@@ -5,6 +5,7 @@ set -e
 # $1: tag of the docker image
 # $2: Path to Jenkinsfile
 # $3: (Optional) Docker options
+# $4: (Optional) Jenksinfile-runner options
```

Comment:
What if the execution set "$4" and not "$3"? I mean: 
`run_jfr_docker_image "my-tag" "/my/paht/Jenkinsfile" "--no-sandbox"` would be a valid execution but it will consider the Jenkinsfile-runner options as Docker options.
Maybe it's worth to start using EVN VAR to set the optional values?

Reply:
@fcojfernandez  I've run this with $3 just set to "" and it worked properly:

  result=$(run_jfr_docker_image "$jenkinsfile_runner_tag" "$current_directory/test_resources/test_cwp_casc_simple/Jenkinsfile" "" "--no-sandbox")

What would you like to do?  Just set JFR_OPTS or something like that?

Reply:
I think it would be better to have multiple functions with different parameters (like overriding the functions). The ENV VARs can be a bit confusing.
Please, update the tests if necessary.